CRM/Extension/Downloader.php

```diff
@@ -197,18 +197,24 @@ public function extractFiles($key, $zipFile) {
       $extractedZipPath = $this->tmpDir . DIRECTORY_SEPARATOR . $zipSubDir;
       if (is_dir($extractedZipPath)) {
         if (!CRM_Utils_File::cleanDir($extractedZipPath, TRUE, FALSE)) {
-          CRM_Core_Session::setStatus(ts('Unable to extract the extension: %1 cannot be cleared', [1 => $extractedZipPath]), ts('Installation Error'), 'error');
+          $errorMessage = ts('Unable to extract the extension: %1 cannot be cleared', [1 => $extractedZipPath]);
```

Comment:
@mattwire there is a tension here between best practice for UI message & for log

For log purposes I think it would be better to do

```
\Civi::log()->error('Unable to extract the extension {extension} to {path}', [
  'extension' => a,
' path' => b,
]);
```
https://docs.civicrm.org/dev/en/latest/framework/logging/